manual/manual_intro.rst

```diff
@@ -71,24 +71,24 @@ Installation
 .. license
 .. pip install pytket
 
-Tket is currently available through its pythonic realisation ``pytket``, which is freely available under the Apache 2 license. To install using the ``pip`` package manager, just run ``pip install pytket`` from your terminal. Each extension module can also be installed similarly as ``pip install pytket_X``, e.g. ``pip install pytket_qiskit``.
+Tket is currently available through its pythonic realisation ``pytket``, which is freely available under the Apache 2 license. To install using the ``pip`` package manager, just run ``pip install pytket`` from your terminal. Each extension module can also be installed similarly as ``pip install pytket-X``, e.g. ``pip install pytket-qiskit``.
 
 .. Link to troubleshooting
 
-``pytket`` is available for python versions 3.8-10 on Linux, MacOS, and Windows. For any difficulties with installation, please consult our `troubleshooting <https://tket.quantinuum.com/api-docs/install.html>`_ page.
```

Comment:
removing reference to the specific python versions as it will inevitably get out of date with what we've written in other places 

As evidenced by 3.8-3.10 when it should be 3.10-3.12